README.md

```diff
@@ -79,76 +90,118 @@ dependencies and compiling is simple.  Just run:
 
     sbt compile
 
-## Running
+## Memory requirements
 
-You can run `openie` with sbt or create a stand-alone jar.  `openie` requires
-substantial memory.  `sbt` is configured to use these options by default:
+`openie` requires substantial memory.  `sbt` is configured to use these options by default:
 
-   -Xmx4G -XX:+UseConcMarkSweepGC
+    -Xmx4G -XX:+UseConcMarkSweepGC
 
-### Running with sbt
+## Usage
 
-    sbt 'run-main edu.knowitall.openie.OpenIECli'
+### Command Line Interface
 
-### Running from a stand-alone jar.
+Check out this project and use `sbt` to run the CLI:
 
-First create the stand-alone jar.
+    % git clone git@github.com:allenai/openie-standalone.git
+    % cd openie-standalone
+    % sbt "runMain edu.knowitall.openie.OpenIECli"
+    ...
+    [info] * * * * * * * * * * * * *
+    [info] * OpenIE 4.1.x is ready *
+    [info] * * * * * * * * * * * * *
 
-    sbt clean compile assembly
+`openie` takes a number of command line arguments.  To see them all run
+`sbt "runMain edu.knowitall.openie.OpenIECli --usage"`. Of particular interest are
+`--ignore-errors` which continues running even if an exception is encountered, `--binary` which gives the binary(triples) output and `--split` which splits the input document text into sentences.
```

Comment:
It's kind of long and will change over time. (For example, I removed docs about first/last argument, because they've been replaced with `--input-file` and `--output-file` arguments.) I'd like to keep this reference to `--usage` as is.